docs/rest_api.md

```diff
@@ -80,11 +80,14 @@ wsk namespace list -v
 ```
 ```
 REQUEST:
-[GET]	https://openwhisk.ng.bluemix.net/api/v1/namespaces
+[GET]  https://openwhisk.ng.bluemix.net/api/v1/namespaces
 Req Headers
 {
   "Authorization": [
     "Basic XXXYYYY"
+  ],
+  "User-Agent": [
```

Comment:
I backed out the changes referencing the rest api requiring User-Agent, but this is an appropriate doc update for the CLI changes in my mind.